plugins/account_history_plugin/include/eosio/account_history_plugin/account_transaction_history_object.hpp

```diff
@@ -40,13 +40,12 @@ using account_transaction_history_multi_index = chainbase::shared_multi_index_co
    account_transaction_history_object,
    indexed_by<
       ordered_unique<tag<by_id>, BOOST_MULTI_INDEX_MEMBER(account_transaction_history_object, account_transaction_history_object::id_type, id)>,
-      hashed_non_unique<tag<by_account_name>, BOOST_MULTI_INDEX_MEMBER(account_transaction_history_object, account_name, account_name), std::hash<account_name>>,
-      hashed_unique<tag<by_account_name_trx_id>,
+      ordered_non_unique<tag<by_account_name>, BOOST_MULTI_INDEX_MEMBER(account_transaction_history_object, account_name, name)>,
```

Comment:
Even though this isn't touched by contracts, we may as well maintain the best practices for these tables.  This means that all indices need to be deterministic.  In other tables, we have achieved this by only using ordered unique indices and adding in the `id` field as a means to distinguish between two otherwise identical entries.

Reply:
same goes for all the indices.